### DISCUSSION TITLE
Error after editing "Sort Order"

### DISCUSSION THREAD

**Discussion by mariaarcena** [microsoft-search/pnp-modern-search#1740] [Q&A] [answered]
Hello! I went to search if someone had the same error (below) as me when it came to editing the "Sort Order" and it seems someone did back in October https://github.com/microsoft-search/pnp-modern-search/discussions/1406

It seems that it was fixed in the latest release (December, which I have installed) but it seems like it's not fixed since I have the error 😔

Is anyone else having this or is there something I am missing?

I am using Microsoft Edge, Windows 10
[HTTP]:400 - [CorrelationId]:f81419a0-c069-c000-c82d-91f8e590ed4f [Version]:16.0.0.22105
![image](https://user-images.githubusercontent.com/17426994/150615704-6ef6d9ad-d4c9-4f39-a292-9acd50cc6f83.png)

**Comment by mariaarcena** [comment, marked as answer]
Okay...I was searching here https://docs.microsoft.com/en-us/sharepoint/technical-reference/crawled-and-managed-properties-overview

I was trying to sort by "WorkEmail" and it seems that WorkEmail is not sortable according to the link above. Even though the article doesn't include Sharepoint Online, it also applies to Sharepoint Online when it comes to crawled and managed proprties...?

So I ended up choosing "LastName" and now it works! Sorry for submitting my question above! 😅

But I do have a question if someone can answer...so I read that by default it sorts by ranking? Like how does that work? Is that just by the api that is being used and in combination looking at the "Rank" property? My team mates are asking how it ranks and I have no good answer 😅

**Reply by wobba** [reply]
The overview does also mostly apply for Online as well, so you can use it as a reference. You can also go to the schema admin on the tenant or a site and take a look at the settings for different managed properties.

**Reply by mariaarcena** [reply]
@wobba I see I see! Makes sense! Thank you!
